src/fp.rs

```diff
@@ -321,12 +322,14 @@ impl Fp {
     }
 
     #[inline]
+    /// Performs the square root of an element in constant time.
+    ///
+    /// NOTE:
```

Comment:
We could leave this note as comment instead of documentation.

Reply:
I thought it was relevant. As there are multiple `sqrt` methods out there. 
But I can amend if you feel strong against adding it to the doc comments.

Reply:
I'm fine with both TBH